src/server/start.ts

```diff
@@ -25,9 +25,14 @@
 
 import { Server } from '@hapi/hapi'
 import Logger from '@mojaloop/central-services-logger'
+import { Util } from '@mojaloop/central-services-shared'
+import Config from '../shared/config'
+import { initializeInstrumentation } from '../shared/util'
 
 export default async function start (server: Server): Promise<Server> {
   await server.start()
   Logger.info(`thirdparty-api-adapter is running @ ${server.info.uri}`)
+  await Util.Endpoints.initializeCache(Config.ENDPOINT_CACHE_CONFIG)
+  initializeInstrumentation()
```

Comment:
Is there any need for this to be it's own function? It's more readable if not, but I can understand how separating it makes testing easier.

Reply:
Ok moved it from util.ts to here